libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -327,6 +329,83 @@ class PostgresVectorStore extends VectorStore {
 
     return rows;
   }
+
+  async maxMarginalRelevanceSearch(
+    query: string,
+    options?: MaxMarginalRelevanceSearchOptions<this["FilterType"]>,
+    _callbacks?: Callbacks | undefined
```

Comment:
Interface says this isn't optional. We will also ignore callbacks for now
```suggestion
    options: MaxMarginalRelevanceSearchOptions<this["FilterType"]>
```